types/index.d.ts

```diff
@@ -163,5 +163,61 @@ declare global {
              */
             function openWebResource(webResourceName: string, windowOptions?: SizeOptions, data?: string): void;
         }
+        namespace WebApi {
+            /**
+             * Creates a table record
+             * 
+             * See {@link https://docs.microsoft.com/en-us/powerapps/developer/model-driven-apps/clientapi/reference/xrm-webapi/createrecord docs.microsoft.com}
+             * 
+             * @param entityLogicalName Logical name of the table you want to create. For example: "account"
+             * @param data A JSON object defining the columns and values for the new table record.
+             * 
+             */
+            function createRecord(entityLogicalName: string, data: any): Promise<any>;
```

Comment:
Please add an inline type for the promise return type

Reply:
Added